controller/internal/supervisor/iptablesctrl/acls.go

```diff
@@ -1358,6 +1344,32 @@ func (i *Instance) deleteAllContainerChains(appChain, netChain string) error {
 	return nil
 }
 
+// setServiceIPs sets acls for the control plane.
+func (i *Instance) setServiceIPs(serviceIPs []string) error {
+
+	var err error
+	if i.mode == constants.LocalServer || i.mode == constants.Sidecar {
+
+		for _, ips := range serviceIPs {
+			if err = i.ipt.Insert(i.appPacketIPTableContext, i.appPacketIPTableSection, 1,
+				"-d", ips, "-j", "ACCEPT",
+			); err != nil {
+				return fmt.Errorf("Unable to add wutai ip %s as excluded network in output chain:%s", ips, err)
+			}
```

Comment:
Fixed and This is to not block traffic from our backend as we monitor all ports. We added it in the input chain earlier but with iptables 1.6.2=> and with the iptables-restore implementation. Input chain is flushed on every policy update and the exclusion rule we added is gone and the traffic is blocked again. So add it in trireme chains

Reply:
It has to be defined as an excluded network. Not done in Trireme.

Reply:
In other words, the policy or enforcer must declare it as an excluded network. There is zero code change needed in Trireme for that.

Reply:
That may not work in the below cases. The excluded network rules are programmed per PU and if changed in the enforcer profile it will overwrite the existing one on next policy update and we may lose the previously programmed rules. The service IP rules need to be at the top of INPUT or OUTPUT chains.

Reply:
and also if done via excluded network they get programmed on every policy update as pu rules are refreshed. Hence if we add it to the top of INPUT chain/OUTPUT chain when the enforcer (supervisor) starts, it need not be programmed again.